leits/MeetingBar
# Added translation in Japanese

### Status
**READY**

### Description

MeetingBar supported multilingualization, so I added Japanese translation. This is a very important improvement for Japanese people.

## Checklist

### Steps to Test or Reproduce
1. Build an App and Run it
2. Click MeetingBar on Status bar, and click preferences
3. Set preferred language to `Japanese`
4. Restart MeetingBar
5. You can see Japanese translation
